ios/brave-ios/Tests/ClientTests/Helpers/URLBarHelperTests.swift

```diff
@@ -24,16 +24,21 @@ class URLBarHelperTests: XCTestCase {
   }
 
   func testSafeQueryList() {
-    testQuery(list: validQueryList, isSuspicious: false)
+    Task {
+      await testQuery(list: validQueryList, isSuspicious: false)
+    }
   }
 
   func testSuspiciousQueryList() {
-    testQuery(list: suspiciousQueryList, isSuspicious: true)
+    Task {
+      await testQuery(list: suspiciousQueryList, isSuspicious: true)
+    }
   }
```

Comment:
These tests will always pass because scope will end before Task runs, you need to make the tests async

swift
```
func testSuspiciousQueryList() async { 
  await testQuery(list: suspiciousQueryList, isSuspicious: true)
}
```

Reply:
Fix at https://github.com/brave/brave-core/pull/24376/commits/ba1de63fe54c8c87eeca41250a710c8beb3f2453